src/css/custom.scss

```diff
@@ -200,3 +193,143 @@ select[data-testid="example-pairing-select"] {
   font-family: "Font Awesome 6 Brands";
   font-size: x-large;
 }
+
+/* Table Styles */
+.theme-doc-markdown {
```

Comment:
This css is applied throughout the whole repo and for all markdown tables (I think). And because you've hardcoded some widths it makes some tables look weird

https://developers.stellar.org/docs/validators/admin-guide/prerequisites
![Screenshot 2024-11-08 at 8 29 56 PM](https://github.com/user-attachments/assets/b18030f6-08f8-49ae-b259-c878581d6f2d)
vs
![image](https://github.com/user-attachments/assets/2a0d061f-3aa1-4895-aece-c9c84d116d99)

https://developers.stellar.org/docs/data
![image](https://github.com/user-attachments/assets/96dc2006-a277-49dd-a53f-57eb26ed02f4)
vs
![image](https://github.com/user-attachments/assets/1af9937e-9664-4a43-99ef-2a68a838f59b)